### DISCUSSION TITLE
Enable to disable the system fields

### DISCUSSION THREAD

**Discussion by juancarlosjr97** [directus/directus#4680] [Feature Requests]
I have to imagine there was a reason to block the editing of the system fields. However, I am working on a project where all the user fields and entirely unnecessary, only email password and 2FA.

What about allowing to hide the fields from the view but not to edit them?

P.S: Sorry it was already discussed but I could not find any discussion about this

**Comment by rijkvanzanten** [comment]
It makes sense to me, but I'm not sure if that's something we can easily allow. The system field information isn't stored in the database, and therefore isn't user editable 🤔 (this is by design and won't change, as mixing user records and system fields turned out to be a nightmare in migrations and resulted in a lot of people accidentally destroying parts of the system)

**Reply by juancarlosjr97** [reply]
Ok, so if I understood is a no, right?

What about, only allowed to show and hide, not edit the field? We can have the settings by default as visually available but not editable?

**Reply by rijkvanzanten** [reply]
well the problem is the same: we don't store the system fields in the database, so there's no place to save the hidden state 🤔

**Reply by juancarlosjr97** [reply]
What about if we do it haha?

If you like the idea, I could work on that PR, to create that setting and allow only to save the hidden state of the default system collections.

**Reply by rijkvanzanten** [reply]
No lets not go that route yet. Merging those settings between database and static file will be a little tricky to maintain longterm, and i'm not sure about the 80/20ness of this yet

**Reply by juancarlosjr97** [reply]
Cool, lets put a pin on it for a while. Thank you @rijkvanzanten!

**Reply by juancarlosjr97** [reply]
@rijkvanzanten perhaps it is something that could be considered? At least just to hide to reduce the view of unused fields for the user for those system collections?

**Reply by rijkvanzanten** [reply]
Yeah, sure thing, though what I said initially is still the case:

> It makes sense to me, but I'm not sure if that's something we can easily allow. The system field information isn't stored in the database, and therefore isn't user editable

**Reply by philleepflorence** [reply]
> @rijkvanzanten perhaps it is something that could be considered? At least just to hide to reduce the view of unused fields for the user for those system collections?

@juancarlosjr97 - I am looking into extending something like a [Presentation Interface](https://github.com/directus/directus/tree/main/app/src/interfaces) and combining the logic of [Conditional Fields#3151](https://github.com/directus/directus/issues/3151#issuecomment-838480337) to allow Admins some field interface control, any ideas?

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
